salvo/src/lib/benchmark/test_fully_dockerized_benchmark.py

```diff
@@ -2,25 +2,81 @@
 Test the fully dockerized benchmark class
 """
 
-import site
 import pytest
+from unittest import mock
 
 from api.control_pb2 import JobControl
 from src.lib.benchmark.fully_dockerized_benchmark import Benchmark
 
 
 def test_images_only_config():
-  """Test benchmark validation logic."""
-  pass
+  """Test benchmark validation logic.
 
+  Validate that we attempt to run the specified docker image
+  when all parameters are present.
+  """
+  # create a valid configuration defining images only for benchmark
+  job_control = JobControl()
```

Comment:
(optional) You may or may not like the kwargs based proto API which allows slightly less repetitive constructs like:

```python
job_control = JobControl(
    remote=False,
    scavenging_benchmark=True,
)
```

Reply:
Did not know you could do that! Let me work on this.